src/NextFederationPlugin.js

```diff
@@ -348,9 +348,7 @@ function generateRemoteTemplate(url, global) {
   return `promise new Promise(function (resolve, reject) {
     var __webpack_error__ = new Error();
     if (typeof ${global} !== 'undefined') return resolve();
-    __webpack_require__.l(
-      ${JSON.stringify(url)},
-      function (event) {
+    __webpack_require__.l(${JSON.stringify(url)}, (event) => {
```

Comment:
Must use es5 function. Not arrow function. This code is not transpiled by webpack. It's taken verbatim